src/Raven.Server/RavenServer.cs

```diff
@@ -452,6 +453,12 @@ private void UpdateCertificateExpirationAlert()
             }
         }
 
+        private double CalculateDaysToRenewBeforeExpiration(X509Certificate2 serverCertificate)
+        {
+            // 30% of the certificate lifetime
+            return Math.Floor((serverCertificate.NotAfter - serverCertificate.NotBefore).TotalDays * 0.3);
```

Comment:
do we want to have it configurable or leave it with hardcoded 30%?

Reply:
Added.